### ISSUE TITLE
How to avoid overfitting in HRNET training?

### ISSUE THREAD

**Issue by XiaotianJia** [leoxiaobin/deep-high-resolution-net.pytorch#209]
Dear all, 

HRNet is really a good network. Currently, I am training a HRNet model to predict key point for my own 3D data. However, overfitting happened in my training and the loss curve of training and validation is shown as following:

![loss](https://user-images.githubusercontent.com/26128456/94233849-29f07800-ff3b-11ea-9aa9-111cd92b4f42.png)

In addition, for validation data, most of the predicted key points seem to be correct. But I think the prediction may be more precise if there is no overfitting.
Has anyone else observed overfitting in the HRNET training? Is there some suggestions to reduce the overfitting?

Any suggestion is appreciated.
